src/main/java/com/spotify/reaper/service/SegmentRunner.java

```diff
@@ -134,6 +142,12 @@ public void postponeCurrentSegment() {
       RepairSegment segment = context.storage.getRepairSegment(repairRunner.getRepairRunId(), segmentId).get();
       postpone(context, segment, context.storage.getRepairUnit(segment.getRepairUnitId()));
     }
+    
+    try {
+      Thread.sleep(SLEEP_TIME_AFTER_POSTPONE_IN_MS);
+    } catch (InterruptedException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)